src/View/Helper/FormHelper.php

```diff
@@ -191,6 +191,14 @@ public function input($fieldName, array $options = [])
         $options = $this->_parseOptions($fieldName, $options);
         $reset = $this->templates();
 
+        $templater = $this->templater();
+        if (!empty($options['templates'])) {
+            $templater->push();
+            $templateMethod = is_string($options['templates']) ? 'load' : 'add';
+            $templater->{$templateMethod}($options['templates']);
+        }
+        unset($options['templates']);
```

Comment:
Yes, !empty() and `[]` or any non-nullish value might end up being outputted in the input one maybe.